core/trino-spi/src/main/java/io/trino/spi/eventlistener/ColumnInfo.java

```diff
@@ -27,6 +28,7 @@
     private final List<String> masks;
 
     @JsonCreator
+    @Unstable
```

Comment:
undo?

Reply:
well, i added this here intentionally. Why would you want me to undo here?